tests/postgres_tests/test_aggregates.py

```diff
@@ -233,6 +233,44 @@ def test_string_agg_array_agg_ordering_in_subquery(self):
                 ).order_by('char_field').values_list('char_field', 'agg')
                 self.assertEqual(list(values), expected_result)
 
+    def test_string_agg_filter(self):
+        values = AggregateTestModel.objects.aggregate(
+            stringagg=StringAgg(
+                'char_field',
+                delimiter=';',
+                filter=Q(char_field__endswith='3')
+            )
+        )
+        self.assertEqual(values, {'stringagg': 'Foo3'})
```

Comment:
This test doesn't fail with or without the patch applied so it's likely unnecessary.

Reply:
You're right, but since the `StringAgg` was not tested in combination with the `filter` option (and therefore not tested in combination with the `OrderableAggMixin` class), I thought it to be a new non-regression test for an untested case.
But if you insist, I'll remove it immediately :)

Reply:
I'll let the committers decide here, I guess there's no harm in adding this test in a separate commit.